From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: zhang sanshan <pete.zhang@nxp.com>
Date: Tue, 30 Jun 2020 17:00:10 +0800
Subject: ANDROID: GKI: make VIDEOBUF2_DMA_CONTIG under
 GKI_HIDDEN_MEDIA_CONFIGS

it use vb2_dma_contig_memops as default mem_ops in csi driver
drivers/media/platform/mxc/capture/mx6s_capture.c still not upstream to linux community.
q->mem_ops = &vb2_dma_contig_memops;

mem_ops is need in videobuf2-core.c to operate dma buffer.
videobuf2-dma-contig.c is common code which have no hardware involved

Bug: 160195378
Signed-off-by: zhang sanshan <pete.zhang@nxp.com>
Change-Id: Ib084ff96bd4f92aa36f8abb8d4b62a0e9be62e6c
---
 init/Kconfig.gki | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init/Kconfig.gki b/init/Kconfig.gki
--- a/init/Kconfig.gki
+++ b/init/Kconfig.gki
@@ -93,6 +93,7 @@ config GKI_HIDDEN_MEDIA_CONFIGS
 	select CEC_CORE
 	select CEC_NOTIFIER
 	select CEC_PIN
+	select VIDEOBUF2_DMA_CONTIG
 	help
 	  Dummy config option used to enable hidden media configs.
 	  These are normally selected implicitly when a module
